### ISSUE TITLE
Add support for "onImportComplete" notification options

### ISSUE THREAD

**Issue by kitizz** [devopsarr/ansible-collection-sonarr#82]
Sonarr recently added a new option for notifications: [On Import Complete](https://github.com/Sonarr/Sonarr/pull/6937).
In the Sonarr API, this is `onImportComplete`.
Also added to sonarr-py [here](https://github.com/devopsarr/sonarr-py/commit/df2a25fc2f3a17c92616f576912985b9a191a911#diff-adde31eb619e5f0ff6825a76bf214533e5bbb9f79365062306502cbc44865a28R70).

The ansible module should be updated to support this new option.

I'm happy to do it! I'll brb in a couple of hours...

**Comment by kitizz** [comment]
Of interest, how do you handle compatibilities between different versions of the ansible module and the sonarr-py module?

**Comment by Fuochi** [comment]
Hi @kitizz, thanks for opening the issue and the PR.
Right now I'm fetching dependency updates through renovate, and I'm using the [requirements.txt](https://github.com/devopsarr/ansible-collection-sonarr/blob/bd07a9470328a4aa7933fc3f63c81de5f84a432c/requirements.txt) to identify the used sdk version.

However, I manage to check updates just from time to time.

Similarly, I'm fetching Sonarr dependency updates on the SDK through renovate.

If you have suggestions on this matter, I'm all hears.

**Comment by kitizz** [comment]
Ah gotcha! Might be good to call out in the main README a specific sonarr-py version to use for that version of the ansible collection. I ran into an issue when I first got it going where the latest released version of this ansible collection isn't compatible with the latest released version of sonarr-py.

Anyway, apologies, this is off topic. Maybe sometime this week I'll submit a pull request for that too ^_^
